include/pytorch/tokenizers/tiktoken.h

```diff
@@ -23,6 +23,12 @@
 #include <pytorch/tokenizers/result.h>
 #include <pytorch/tokenizers/tokenizer.h>
 
+#ifdef _WIN32
+// ssize_t isn't available on Windows. Alias it to the Windows SSIZE_T value.
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
```

Comment:
In hindsight, this should probably be typedefed as TK_SSIZE_T or something similar as this is a public header. I'll update this before landing

Reply:
I added a new compiler.h header for platform and toolchain specific defs, following the pattern used in ET. I was originally going to introduce a new typedef or macro for TK_SSIZE_T, but looking at other projects, it seems like just directly typedefing to size_t is common, so I've done that. If anyone has strong opinions, I'm happy to change it.